blog-post.html

```diff
@@ -2,11 +2,92 @@
 <html>
   <head>
     <title>Blog Post</title>
+    <style>
+      @import "index.css";
+    </style>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
   </head>
   <body>
-    <section id="header"></section>
-    <section id="main"></section>
+
+    <section id="header">
+      <h1><a class="header-link" href="index.html">POST ONLY</a></h1>
+      <H6>A PAGE WITH JUST A BLOG POST CONTENT.</H6>
+    </section>
+<br>
+<br>
+    <section id="main4">

```

Comment:
Try to use better variables such as "blogPostMain" or something rather then just "main4"